reporter/otlp_reporter_test.go

```diff
@@ -69,6 +72,12 @@ func TestGetSampleAttributes(t *testing.T) {
 						Value: &common.AnyValue_StringValue{StringValue: "apmServiceName1"},
 					},
 				},
+				{
+					Key: "process.pid",
+					Value: &common.AnyValue{
+						Value: &common.AnyValue_StringValue{StringValue: "pid1"},
```

Comment:
Reporting a non-numerical value for `process.pid` should result in an error. The [OTel Semantic Convention](https://opentelemetry.io/docs/specs/semconv/attributes-registry/process/) expects also a numeric value for `process.pid`.